magma/lib/magma/validation/attribute.rb

```diff
@@ -58,16 +58,24 @@ def validation_object
         private
 
         def format_error(value)
-          if @attribute.format_hint
-            "On #{@attribute.name}, '#{value}' should be like '#{@attribute.format_hint}'."
+          if link_attribute.format_hint
```

Comment:
Is this intended change?

Reply:
I think this is correct? The error comes from link_validate, which actually just runs the validation for the link model identifier. If there is a format hint for this validation, it is likely on that `link_attribute`, and not on our `@attribute` (which is probably like a `parent` or something).